packages/yew-router-macro/tests/routable_derive/valid-pass.rs

```diff
@@ -0,0 +1,12 @@
+#[derive(Debug, PartialEq, yew_router::Routable)]
```

Comment:
You know the drill, please test without the implicit prelude (`#![no_implicit_prelude]`) and without bringing any types into scope. You should find that you still have a lot of cases where you're relying on it in the generated code.